source/macos/AdaptiveCards/AdaptiveCards/Views/ACRView.swift

```diff
@@ -179,12 +180,14 @@ class ACRView: ACRColumnView {
             }
             
             // toggle the last padding if all other
-            if facade?.visibilityManagerAllStretchableViewsHidden() ?? false {
-                facade?.visibilityManagerSetLastStretchableView(isHidden: !isHide)
-            } else {
-                facade?.visibilityManagerSetLastStretchableView(isHidden: isHide)
-            }
+            facade?.visibilityManagerSetLastStretchableView(isHidden: !(facade?.visibilityManagerAllStretchableViewsHidden() ?? false))
+            facadeArray.append(facade)
```

Comment:
`visibilityManagerSetLastStretchableView(isHidden: isHide)` is depend on **isHide** variable.
can you please check?